### DISCUSSION TITLE
How do I import `asKeyStore()` ?

### DISCUSSION THREAD

**Discussion by douglance** [panva/jose#168] [Q&A] [answered]
I have read the entirety of the docs, searched through the codebase, and tried many combinations of import statements to no avail. 

I'd like to import `asKeyStore()` from `jose`. It looks like it isn't exported. 

I am trying to inject a keystore rather than relying on `jwks_uri` to fetch it, because the service I'm integrating with doesn't use that feature.

**Comment by panva** [comment, marked as answer]
There is no keystore in jose@3. You can just build your own [getKeyFunction](https://github.com/panva/jose/blob/main/docs/interfaces/_jwt_verify_.jwtverifygetkey.md) to pass to [verify](https://github.com/panva/jose/blob/main/docs/functions/_jwt_verify_.jwtverify.md) instead of a KeyLike where you select the key yourself.

**Reply by panva** [reply]
In other words - what you see in the docs is all there is in v3.x

**Reply by douglance** [reply]
Sorry--this might be better in the `openid-client` repo. I'll try to phrase it better there.
